usb_autorunner_profiles/deploy/package_resources.sh

```diff
@@ -27,9 +27,9 @@ mkdir -p $RESOURCES_DIR
 
 # Fetch distro
 echo "⚙️ Download $DISTRO_NAME distro..."
-wget $DISTRO_URL -O $BUILD_DIR/bahmni-distro-c2c.zip
-mkdir -p $RESOURCES_DIR/distro
-unzip $BUILD_DIR/bahmni-distro-c2c.zip -d $RESOURCES_DIR/distro
+mvn org.apache.maven.plugins:maven-dependency-plugin:3.2.0:get -DremoteRepositories=https://nexus.mekomsolutions.net/repository/maven-public -Dartifact=$ARTIFACT_GROUP:bahmni-distro-$DISTRO_GROUP:$DISTRO_VERSION:zip -Dtransitive=false --legacy-local-repository
+mvn org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy -Dartifact=$ARTIFACT_GROUP:bahmni-distro-$DISTRO_GROUP:$DISTRO_VERSION:zip -DoutputDirectory=$RESOURCES_DIR/distro
```

Comment:
Seems like it's missing the unzip step.
Let's actually replace by mvn dependency:unpack which copies and unzips the artifact in 1 step.